src/plugins/data/server/search/collectors/register.ts

```diff
@@ -10,18 +10,22 @@ import { PluginInitializerContext } from 'kibana/server';
 import { UsageCollectionSetup } from '../../../../usage_collection/server';
 import { fetchProvider } from './fetch';
 
-export interface Usage {
+export interface CollectedUsage {
   successCount: number;
   errorCount: number;
-  averageDuration: number | null;
+  totalDuration: number;
 }
 
+export type ReportedUsage = Omit<CollectedUsage, 'totalDuration'> & {
+  averageDuration: number;
+};
+
 export async function registerUsageCollector(
   usageCollection: UsageCollectionSetup,
   context: PluginInitializerContext
 ) {
   try {
-    const collector = usageCollection.makeUsageCollector<Usage>({
+    const collector = usageCollection.makeUsageCollector<ReportedUsage>({
```

Comment:
From a telemetry POV, the shape of the data that's being reported (returning from the `fetch` method) hasn't changed so nothing further is needed for that and we're good on that front.

Reply:
It looks like the slight type change _did_ cause the schema check to fail. I don't quite know why because the final document's schema hasn't changed.
Please run `node scripts/telemetry_check --fix` to resolve the failing test.